scss/prototype/_display.scss

```diff
@@ -0,0 +1,50 @@
+// Foundation for Sites by ZURB
+// foundation.zurb.com
+// Licensed under MIT Open Source
+
+////
+/// @group prototype-display
+////
+
+/// Responsive breakpoints for display, defaulted to global-prototype-responsive-breakpoints.
+/// @type Boolean
+$prototype-display-responsive-breakpoints: $global-prototype-responsive-breakpoints !default;
+
+@mixin foundation-prototype-display {
+  @each $display in (none, inline, inline-block, block, table, table-cell) {
```

Comment:
Hmmmnnn,,,,, @natewiebe13 @IllusionPerdu but,
Where where do i add these and where i leave??
In `display`, `list-style-type` , `text-decoration`, `text-transformation` it makes sense because there it should be customized.

But what about `float` and `text-alignment`  
- Float has (left, right)
- Text alignment has (left, right, center, justify)

Will it make sense to add in these two also, because you don't want user to customize that??